client/src/test/App.test.tsx

```
@@ -1,24 +1,36 @@
+vi.mock('../components/ErrorComponent', () => ({
+  default: () => <div>Page Not Found</div>,
+}));
+
+// Mock AuthProvider to avoid Firebase initialization
+vi.mock('../Hooks/contextApi/UserContext', () => ({
+  default: ({ children }: { children: React.ReactNode }) => <div>{children}</div>,
+}));
+
+// Mock Toaster to avoid side effects
+vi.mock('react-hot-toast', () => ({
+  Toaster: () => <div data-testid="toaster"></div>,
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)